Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -125,30 +125,35 @@ export default function widgetBehavior(publicAPI, model) {
       return macro.VOID;
     }
     const moveHandle = model.widgetState.getMoveHandle();
+    moveHandle.setVisible(false);
     if (
       model.activeState === model.widgetState.getMoveHandle() &&
-      nbHandle1 === 0
+      nbHandles === 0
```

Comment:
compute nbHandles each time you enter handleLeftButtonPress (or store it in the state).

I guess it should be possible by testing handle visibility or origin validity (if null/undefined or set to a valid position)